controllers/oauth2client_controller.go

```diff
@@ -22,13 +23,18 @@ import (
 )
 
 const (
-	ClientIDKey     = "client_id"
-	ClientSecretKey = "client_secret"
-	FinalizerName   = "finalizer.ory.hydra.sh"
+	DefaultClientID  = "CLIENT_ID"
+	DefaultSecretKey = "CLIENT_SECRET"
+	FinalizerName    = "finalizer.ory.hydra.sh"
 
 	DefaultNamespace = "default"
 )
 
+var (
+	ClientIDKey     = DefaultClientID
```

Comment:
The idea is pretty simple and I see no reason to why not merge it, but need one little thing first ;) As we're adding new config options, with Envs, please update the README accordingly!